jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -20,120 +20,81 @@ def buildExclusions = [
   // Machine Label,                   Build Type,  Node Version
 
   // Linux -------------------------------------------------
-  [ /^centos7-(arm)?(64|32)-gcc48/,anyType,     gte(10) ],
-  [ /^centos7-(arm)?(64|32)-gcc6/, anyType,     lt(10)  ],
-  [ /^centos7-(arm)?(64|32)-gcc6/, anyType,     gte(14) ],    // 14.x: gcc6 builds stop
-  [ /^centos7-(arm)?(64)-gcc8/,       anyType,     lt(14)  ], // 14.x: gcc8 builds start
+  [ /^centos7-(arm)?(64|32)-gcc48/,   anyType,     gte(16) ],
+  [ /^centos7-(arm)?(64|32)-gcc6/,    anyType,     gte(16) ],    // 14.x: gcc6 builds stop
   [ /^centos7-(arm)?(64)-gcc8/,       anyType,     gte(18) ], // 18.x: centos7 builds stop
-  [ /^centos7-64/,                    releaseType, lt(12)  ],
   [ /^centos7-64/,                    anyType,     gte(18) ],
-  [ /debian8-x86/,                    anyType,     gte(10) ], // 32-bit linux for <10 only
-  [ /debian8/,                        anyType,     gte(13) ],
+  [ /debian8-x86/,                    anyType,     gte(16) ], // 32-bit linux for <10 only
```

Comment:
Do we have any debian8-x86 labelled machines (in either CI) left? If they're all gone we could remove completely.

Reply:
I agree, but I prefer to do a second pass removing outdated labels.

Reply:
SGTM